src/atap_widgets/concordance.py

```diff
@@ -127,18 +128,20 @@ def prepare_text_df(
         id_column: The current column name of the unique identifier for each text
             in df. If not given, numeric IDs will be generated for each text.
         language_model: The name of a spacy model like "en_core_web_sm", or a
-            spacy language model instance.
+            spacy language model instance. Defaults to English()
     """
     output = df.copy()
     if id_column is None:
-        output["text_id"] = pd.Series(range(output.shape[0]), dtype=pd.Int64Dtype)
+        output["text_id"] = pd.Series(range(output.shape[0]), dtype=pd.Int64Dtype())
         id_column = "text_id"
     output = output.rename(columns={text_column: "text", id_column: "text_id"})
     output = output.set_index("text_id", drop=False)
 
     if isinstance(language_model, str):
         language_model = spacy.load(language_model)
-    output["spacy_doc"] = output["text"].map(language_model)  # Doc for each line
+    output["spacy_doc"] = [
```

Comment:
This looks like a good way to get some easy performance improvements. Looks like `pipe()` has an `n_process` argument, have you checked whether we get additional improvements using multiple processors?

Reply:
When I increase this argument the benchmark I'm using takes twice as long to run